testsuite/heif/run.py

```diff
@@ -1,6 +1,9 @@
 #!/usr/bin/env python
 
 imagedir = "ref/"
-files = [ "IMG_7702_small.heic" ]
+files = [ "IMG_7702_small.heic", "Chimera-AV1-8bit-162.avif" ]
 for f in files:
     command = command + info_command (os.path.join(imagedir, f))
+
+# AVIF file is expected to fail with libheif without AV1 decoder
+failureok = 1
```

Comment:
Let's remove this (failureok=1 masks any real problems, so what's the point of the test?) and just ensure that CI and any developers who run tests and care about this will have libheif >= 1.7.

Reply:
brew currently ships libheif without AV1 enabled (I have sent a PR for that too) so the macOS tests fail. CI/developers will have to make sure that they have libheif >1.7 with AV1 enabled. Is that OK?

Reply:
I think that's fine.

FWIW, Brew currently ships libheif 1.10 and I found that it's utterly broken on MacOS when writing heif file. I reported the problem and pointed them to the exact commit where it started failing. Awaiting a resolution. But one more problem with the current Brew default build of libheif is a drop in the bucket at the moment.